src/bootstrap/dist.rs

```diff
@@ -1451,7 +1430,7 @@ impl Step for Extended {
 
         let xform = |p: &Path| {
             let mut contents = t!(fs::read_to_string(p));
-            for tool in &["rust-demangler", "rust-analyzer", "miri", "rustfmt"] {
+            for tool in &["rust-demangler", "rust-analyzer", "rustfmt"] {
```

Comment:
Can you go ahead and remove `rust-analyzer` and `rustfmt` while you're at it? they're both subtrees now too :)

Reply:
🤦 yea, will do